src/main/java/org/alliancegenome/curation_api/model/ingest/dto/AffectedGenomicModelDTO.java

```diff
@@ -6,10 +6,13 @@
 import lombok.EqualsAndHashCode;
 import org.alliancegenome.curation_api.constants.LinkMLSchemaConstants;
 import org.alliancegenome.curation_api.interfaces.AGRCurationSchemaVersion;
+import org.alliancegenome.curation_api.model.ingest.dto.slotAnnotions.SecondaryIdSlotAnnotationDTO;
 import org.alliancegenome.curation_api.view.View;
 
 import java.util.List;
 
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.RedundantImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Duplicate import to line 12 - java.util.List.
<!-- __reviewdog__:ChA1NjY4OTdlYzJlYTc4NzNlEgpjaGVja3N0eWxl -->